### PR TITLE
qemu: build using buildroot

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Change-Id: Ib9ba214170552ddbc041610c416de1648cf34cae
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
https://github.com/OP-TEE/build/pull/235

File: default.xml [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
-        <project path="gen_rootfs"         name="linaro-swg/gen_rootfs.git" />
         <project path="linux"              name="linaro-swg/linux.git" revision="optee"/>
         <project path="optee_benchmark"    name="linaro-swg/optee_benchmark.git"/>
         <project path="optee_examples"     name="linaro-swg/optee_examples.git" />
         <project path="soc_term"           name="linaro-swg/soc_term.git" />
 
         <!-- Misc gits -->
-        <project path="busybox"            name="mirror/busybox.git" />
+	<!-- buildroot tag 2017.11 -->
+        <project path="buildroot" name="buildroot/buildroot.git" revision="9dd76697ccd2bac65a78fd7687899e1c9ca14465" />
```
Comment (jforissier) [inline default.xml]: nit: align `name`

File: default.xml [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
         <project path="bios_qemu_tz_arm"   name="linaro-swg/bios_qemu_tz_arm.git" />
-        <project path="gen_rootfs"         name="linaro-swg/gen_rootfs.git" />
         <project path="linux"              name="linaro-swg/linux.git" revision="optee"/>
         <project path="optee_benchmark"    name="linaro-swg/optee_benchmark.git"/>
         <project path="optee_examples"     name="linaro-swg/optee_examples.git" />
         <project path="soc_term"           name="linaro-swg/soc_term.git" />
 
         <!-- Misc gits -->
-        <project path="busybox"            name="mirror/busybox.git" />
+	<!-- buildroot tag 2017.11 -->
```
Comment (jforissier) [inline default.xml]: Would `revision="refs/tags/2017.11"` work, then?

Comment (jenswi-linaro) [reply default.xml]: Seems to work

**Comment by jenswi-linaro** [comment]
Update

**Comment by jforissier** [comment]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
Tag applied

**Pull request merged**